pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -284,6 +284,11 @@ func SetupLibvirt() error {
 		return err
 	}
 
+	_, err = os.Stat("/dev/vhost-net")
```

Comment:
Why is that check here? If `/dev/vhost-net` is present on the node or not should be determined by the device plugin and not by virt-launcher. I think that you can remove that.

Reply:
My bad, thanks.